### ISSUE TITLE
bubble explanation

### ISSUE THREAD

**Issue by MayaGans** [JohnCoene/javascript-for-r-comments#2]
> bubble (Fay 2020) by Colin Fay is a more recent R package, still under heady development but very promising: it lets one run node.js code in R, comes with an interactive node REPL, the ability to install npm packages, and even an R markdown engine. It’s similar to V8 in many ways.

The book has already made the caveat that this wont be a an JS tutorial. That said, maybe talk briefly about what node.js is here (and explain npm too)?

**Comment by MayaGans** [comment]
.....I just skimmed the chapter names instead of reading straight through and see you do cover this later. Maybe sentence in chapter one that says something like this is a high level map of the book and we'll address all terminology later? Maybe you can include internal links so the V8 section can take you to later in the book, and the quoted section above takes you to the NPM section?

**Comment by JohnCoene** [comment]
Yes, good point I should briefly introduce these topics/terms when they first appear in the book.
